docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -18,77 +17,56 @@
 
 
 
-Model Optimizer is a cross-platform command-line tool that facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
-
-To use it, you need a pre-trained deep learning model in one of the supported formats: TensorFlow, PyTorch, PaddlePaddle, TensorFlow Lite, MXNet, Caffe, Kaldi, or ONNX. Model Optimizer converts the model to the OpenVINO Intermediate Representation format (IR), which you can infer later with :doc:`OpenVINO™ Runtime <openvino_docs_OV_UG_OV_Runtime_User_Guide>`.
-
-Note that Model Optimizer does not infer models.
-
-The figure below illustrates the typical workflow for deploying a trained deep learning model:
-
-.. image:: _static/images/BASIC_FLOW_MO_simplified.svg
-
-where IR is a pair of files describing the model:
-
-* ``.xml`` - Describes the network topology.
-
-* ``.bin`` - Contains the weights and biases binary data.
-
-The OpenVINO IR can be additionally optimized for inference by :doc:`Post-training optimization <pot_introduction>` that applies post-training quantization methods.
```

Comment:
It's only been removed, as requested in JIRA ticket: 109837, by @popovaan 

I will reinstate the removed content and apply the relevant changes for `Model Optimizer` only.

Reply:
I asked to remove it here because I expected that this information will be in "Model Preparation" section which introduces different conversion tools.

Reply:
> I asked to remove it here because I expected that this information will be in "Model Preparation" section which introduces different conversion API's (convert_model, read_model and MO cli tool).

Forgive me, but the changes I have made were premised on clear-cut instructions provided in the attachments found in the 109837 Jira ticket, not expectations.

Reply:
No worries, just add this to "Model Preparation".

Reply:
Have added the text to `Model Preparation` article with the [11c4f5172d](https://github.com/sgolebiewski-intel/openvino/commit/11c4f5172dbb75201cf56adb321e6c3f9bcc97b0) commit.